source/opt/passes.h

```diff
@@ -74,6 +75,15 @@ class EliminateDeadConstantPass : public Pass {
   bool Process(ir::Module*) override;
 };
 
+// The optimization pass to deduplicate the constants. Front-end constants with
+// same value will be unified and only one constants will be kept for each
```

Comment:
A little bit not clear for me here is about the same floating point values but with different bit patterns. Are we also unifying them?